src/api/loadPage.js

```diff
@@ -0,0 +1,22 @@
+import axios from "axios";
+import URLS from "../config/constants/url";
+import { useState, useEffect } from "react";
```

Comment:
If we're talking about imports, they separated by 3 categories:
```
libraries

absolute paths

relative paths
```
Example:
```
import axios from "axios";
import { useState, useEffect } from "react";

import URLS from "config/constants/url";
```
1. Setup absolute paths for config folder
2. Sort imports by category